ibmcloudshellv1/ibm_cloud_shell_v1.go

```diff
@@ -0,0 +1,658 @@
+/**
+ * (C) Copyright IBM Corp. 2021.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * IBM OpenAPI SDK Code Generator Version: 3.33.0-caf29bd0-20210603-225214
+ */
+
+// Package ibmcloudshellv1 : Operations and models for the IbmCloudShellV1 service
+package ibmcloudshellv1
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"reflect"
+	"time"
+
+	"github.com/IBM/go-sdk-core/v5/core"
+	common "github.com/IBM/platform-services-go-sdk/common"
+)
+
+// IbmCloudShellV1 : API docs for IBM Cloud Shell repository
+//
+// Version: 1.0
+type IbmCloudShellV1 struct {
+	Service *core.BaseService
+}
+
+// DefaultServiceURL is the default URL to make service requests to.
+const DefaultServiceURL = "https://api.shell.test.cloud.ibm.com"
```

Comment:
We should use the production version of the API definition when generating the SDK code, which should also use the production service URL.

Reply:
Done